tensorflow/core/util/use_cudnn.h

```diff
@@ -39,6 +39,12 @@ FP16ConvMode CudnnConvComputeMode();
 bool DebugCudnnRnn();
 bool DebugCudnnRnnUseTensorOps();
 int64 DebugCudnnRnnAlgo();
+
+// Return true if the filter is 1x1, 3x3, 5x5 or 7x7. This function is used
```

Comment:
Google C++ style says function comments needs to be descriptive, not imperative: https://google.github.io/styleguide/cppguide.html#Function_Comments

Reply:
Done. PTAL.